src/main/java/seedu/address/logic/parser/ArgumentMultimap.java

```diff
@@ -36,11 +36,11 @@ public void put(Prefix prefix, String argValue) {
     }
 
     /**
-     * Returns the last value of {@code prefix}.
+     * Returns the last value of {@code prefix} or the default value if it does not exist.
      */
-    public Optional<String> getValue(Prefix prefix) {
+    public String getValueOrDefault(Prefix prefix, String defaultString) {
         List<String> values = getAllValues(prefix);
```

Comment:
Needs to account for cases where values contains null, empty or whitespace-only strings.